src/apy/config.py

```diff
@@ -2,15 +2,16 @@
 import json
 import os
 from pathlib import Path
+from typing import Optional, Any
 
 
-def get_base_path():
+def get_base_path() -> Optional[str]:
     # If base_path not defined: Look in environment variables
-    if path := os.environ.get("APY_BASE"):
-        return path
+    if path_as_str := os.environ.get("APY_BASE"):
+        return path_as_str
 
-    if path := os.environ.get("ANKI_BASE"):
-        return path
+    if path_as_str := os.environ.get("ANKI_BASE"):
```

Comment:
mypy complains here because in the code below `path` changes its type from `str` to `pathlib.Path`.

Reply:
It's unclear what you mean. Will mypy complain after these changes or was this particular change necessary _because_ mypy complained?

Reply:
mypy was complaining about the previous version of the code. 

All the changes I did in the PR were solely to please mypy. No change in behavior.

Just to clarify what the problem was in the previous version of the code:

* the variable `path` starts off as a string
* down the function, it becomes a `pathlib.Path`
* mypy doesn't allow a variable to change its type and complained